mastering-plone/dexterity_2.rst

```diff
@@ -275,7 +294,8 @@ Alternatively you also select which upgrade steps to run like this:
 Add a browserlayer
 ------------------
 
-A browserlayer is another such marker interface. Browserlayers allow us to easily enable and disable views and other site functionality based on installed add-ons and themes.
+A browserlayer is another such marker interface, but this time on the request.
```

Comment:
Is "browserlayer" an inline literal or a term? If literal, then double backticks needed throughout.

Reply:
term